gradle.properties

```diff
@@ -9,7 +9,7 @@
 #
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx2G -XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-Xmx8G -XX:MaxMetaspaceSize=1G
```

Comment:
I have a beefier debugging machine, but we can reset this down if folks want. This helped speed up my build times quite a bit.

Reply:
[This](https://discuss.gradle.org/t/precedence-of-gradle-properties/41366) might be interesting for you. I'd prefer to keep the lower defaults in this repo.

Reply:
Definitely I'll remove these changes when I can get back on my computer.